src/vs/workbench/parts/extensions/electron-browser/extensionsViewlet.ts

```diff
@@ -294,6 +298,54 @@ export class ExtensionsViewlet extends PersistentViewsViewlet implements IExtens
 		}
 	}
 
+	private hookDragAndDropHandler(target: HTMLElement): void {
```

Comment:
Not needed to have a util class for DnD handler. You can just listen to events directly.